demo-cat-gallery/src/main/java/com/google/android/flexbox/apps/catgallery/perf/TextViewMeasurementBuilder.kt

```diff
@@ -0,0 +1,92 @@
+package com.google.android.flexbox.apps.catgallery.perf
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import android.graphics.Color
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.Base64
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import com.example.stretchplayground.CustomImageView
+import com.example.stretchplayground.CustomTextView
+import com.example.stretchplayground.CustomWrongFlexboxLayout
+import com.google.android.flexbox.AlignSelf
+import com.google.android.flexbox.FlexDirection
+import com.google.android.flexbox.FlexWrap
+import com.google.android.flexbox.FlexboxLayout
+
+class TextViewMeasurementBuilder(
+    private val context: Context) {
+
+  private val imageBase64 =
+      "iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQ" +
+          "UAAABqSURBVHgB7ZCxDYBADAMtsRAbfjZgBDwKm8EjUXyTwsgRBX+Su+SsBJh8zQI/7Fl7DhTAnvNJwAwH+Z0dRvgPeUBrpypXTiTEtygL" +
+          "yqy8GG/lWcnmlGcl4ZRnJdlV9pIGM61SPpaUySf1XBgQU+UofKBZAAAAAElFTkSuQmCC"
+
+  fun build(container: ViewGroup) {
+    val wrapper = defaultContainer()
+    val root = defaultContainer(FlexDirection.ROW, true)
+    val textView = textview()
+
+    root.addView(imageview())
+    root.addView(textView)
+//    root.addView(imageview())
```

Comment:
It is possible to uncomment this line to verify the behavior with ImageView at the beginning and end of the TextView.